README.rst

```diff
@@ -69,7 +69,7 @@ An automated installer is available at `ansible-openwisp2 <https://github.com/op
 Dependencies
 ------------
 
-* Python 2.7 or Python >= 3.5
+* Python 3.5
```

Comment:
I think this should be Python >=3.5

Reply:
Yeah, my bad.. Support of Python3.5+ version.. sorry :smile:

Reply:
Done :smile: